tests/default_test.go

```diff
@@ -0,0 +1,38 @@
+package test
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"runtime"
+	"path/filepath"
+	_ "goMarketer/routers"
```

Comment:
How about separate this 2 groups by blank line (standard library packages and internal packages) and also how about to sort packages by alphabetical order? IMO, it would be easier to read and maintain.

Reply:
Fixed that in https://github.com/andyduong1920/go_marketer/pull/1/commits/37ca365970a8d680f8ee429a370a5a800dbcb02e @gutakk 🚀